### ISSUE TITLE
Connect stereo pairs with other end in single movement

### ISSUE THREAD

**Issue by john-bonachon** [falkTX/Carla#1472]
Hi falk! Have you considered a way to connect both stereo cables to other stereo end in one mouse movement (by holding Shift or Ctrl)?

That way we could save a lot of time, instead of connecting both pairs manually.

**Comment by falkTX** [comment]
Yes there is even a whole PR about this https://github.com/falkTX/Carla/pull/1397

And I am sure a bunch of tickets around the subject.
I am closing this one since it is not new.

FYI I am not doing any new features on Carla for the time being, will revisit the PR and this situation at some point next year.
